doc/adding_new_libraries.md

```diff
@@ -78,18 +78,31 @@ bazel run \
 ci/cloudbuild/build.sh -t checkers-pr
 ```
 
-## Manually add the C++ files to the CMakeLists.txt file
+## Create any custom source files
 
 If the `generator/generator_config.textproto` entry for the service does not
 enumerate the `retryable_status_codes`, you need to manually create the file as
-`google/cloud/$library/internal/<service_name>_retry_traits.h`. Add this file
-**and** any generated C++ files to the `CMakeLists.txt` file. There should
-be markers in the file, search for `EDIT HERE`.
+`google/cloud/$library/internal/<service_name>_retry_traits.h`.
+
+Likewise, for services using streaming operations you may need to implement the
+streaming `*Updater` function. Use `google/cloud/bigquery/streaming.cc` for
+inspiration.
+
+## Validate and Generate the `*.bzl` files
```

Comment:
I believe bazel can use file globs also. It's not a "recommended" approach for bazel, but here we're talking about generated libraries, trying to reduce manual steps, and we're now using globs in cmake also. Something to consider.

Reply:
Ack.  I am not sure which approach would be easier to import into google3. #7642